lib/evmone/eof.cpp

```diff
@@ -13,6 +13,92 @@ namespace
 {
 constexpr uint8_t FORMAT = 0xef;
 constexpr uint8_t MAGIC = 0x00;
+constexpr uint8_t TERMINATOR = 0x00;
+constexpr uint8_t CODE_SECTION = 0x01;
+constexpr uint8_t DATA_SECTION = 0x02;
+
+std::pair<EOF1Header, EOFValidationErrror> validate_eof_headers(bytes_view code) noexcept
```

Comment:
Same here, how about calling this container?

Reply:
Renamed.